src/main/java/module/youth/YouthPlayer.java

```diff
@@ -1272,4 +1271,92 @@ private Integer getTrainingUsefulness(YouthSkillInfo skillInfo) {
         }
         return 0;
     }
+
+    public int calculateRateMyAcademyScore() {
```

Comment:
It might be useful to have unit tests for that method in the future.

Reply:
Created new test class YouthPlayerTests